src/services/apiUtils.ts

```diff
@@ -0,0 +1,40 @@
+import { getUniqueId } from '@patternfly/react-core';
+import { QueryClient } from '@tanstack/react-query';
+
+interface ApiHandlerProps<T> {
+  apiServiceMethod: Function;
```

Comment:
for here and line 11, don't use `Function` as a type as it won't provide any type safety. here you'll probably want something like `(url: string, payload: any) => Promise<AxiosResponse<any, any>>`, for example

Reply:
(with changes proposed in chat, this becomes obsolete)